### ISSUE TITLE
Optimization Suggestion about the Webpage /orc-pdf

### ISSUE THREAD

**Issue by Status-Changer** [Stirling-Tools/Stirling-PDF#728]
I downloaded many .traineddata files from the Tesseract website and added them to .../stirling/trainingData dir.It just simply lists all detected languages in alphabetical order, with no collapse button and no language priority configuration.

![QQ图片20240122144109](https://github.com/Stirling-Tools/Stirling-PDF/assets/93187885/1973cf9e-cbde-4967-b078-6a697c5af23b)

Will you consider turning it into the next feature? Thanks.

**Comment by sbplat** [comment]
Do you mean ranking each language so there's a priority for the OCR? We use OCRmyPDF internally for this feature, and I don't think there's an option to rank them (the `-l` flag hints what languages it should search for).

If I misunderstood your question, please let me know!

**Comment by Status-Changer** [comment]
> Do you mean ranking each language so there's a priority for the OCR? We use OCRmyPDF internally for this feature, and I don't think there's an option to rank them (the `-l` flag hints what languages it should search for).
> 
> If I misunderstood your question, please let me know!

No, I mean setting the priority just on the webpage for the user to choose. For example, if I use Japanese and English in most cases, then I want the list to be shown as 

- Japanese
- English
- ... (Other languages)

instead of the current alphabetical version. And the priority can be set by user, such as sorting by number of uses.

Thanks for your reply.

**Comment by Status-Changer** [comment]
The number of languages used by each user is very limited, maybe this is not necessary for the system -.-

**Comment by sbplat** [comment]
Do you mean to obtain the default locale(s) from the browser and list them in that order?
